base/build.gradle

```diff
@@ -12,6 +12,8 @@ dependencies {
         exclude group: 'com.google.guava'
     }
 
+    api("org.apache.commons:commons-lang3:3.11") // Only for validation
```

Comment:
This should not be used in my opinion as similar methods already in the standard library.

(see `Objects#nonNull`, etc.)

Reply:
I have removed said library